config/manager/kustomization.yaml

```diff
@@ -1,13 +1,5 @@
 resources:
 - manager.yaml
-
-generatorOptions:
-  disableNameSuffixHash: true
-
-configMapGenerator:
-- files:
-  - controller_manager_config.yaml
-  name: manager-config
 apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Reading about changes in the following:
 https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.33.0/

This points to https://book.kubebuilder.io/plugins/available/kustomize-v2

Should the kustomize be moved to v2 ?

Reply:
Kustomize plugin is used under the hood, its in the kubebuilder code https://github.com/kubernetes-sigs/kubebuilder/tree/master/pkg/plugins/common/kustomize/v2 (that Operator SDK uses)

this yaml has the correct version https://github.com/operator-framework/operator-sdk/blob/v1.34.x/testdata/go/v4/memcached-operator/config/manager/kustomization.yaml#L3

Reply:
Ok, thanks. Makes sense.